reference/6/Microsoft.PowerShell.Utility/Export-Clixml.md

```diff
@@ -106,6 +107,49 @@ To import the credential automatically into your script, run the final two comma
 `Import-Clixml` to import the secured credential object into your script. This import eliminates the
 risk of exposing plain-text passwords in your script.
 
+### Example 4: Exporting a credential object on Linux or macOS
+
+In this example, we create a **PSCredential** in the `$Credential` variable using the
+`Get-Credential` cmdlet. Then we use `Export-Clixml` to save the credential to disk.
+
+> [!IMPORTANT]
+> `Export-Clixml` only exports encrypted credentials on Windows. On non-Windows operating systems
+> such as macOS and Linux, credentials are exported as a plain text stored as a Unicode character
+> array. This provides some obfuscation but does not provide encryption.
+
+```powershell
```

Comment:
Do you think we could trim the output for clarity on what we are trying to show? eg. 

```powershell
PS> $Credential = Get-Credential
PowerShell credential request
Enter your credentials.
User: User1
Password for user User1: ********
PS> $Credential | Export-Clixml ./cred2.xml
PS> Get-content  ./cred2.xml

...
<Props>
      <S N="UserName">User1</S>
      <SS N="Password">700061007300730077006f0072006400</SS>
    </Props>
...

PS> 'password' | Format-Hex -Encoding unicode
                 00 01 02 03 04 05 06 07 08 09 0A 0B 0C 0D 0E 0F
0000000000000000 70 00 61 00 73 00 73 00 77 00 6F 00 72 00 64 00 p a s s w o r d
```